ng-client/src/app/core/login.component.ts

```diff
@@ -13,9 +13,8 @@ import {
 } from '../core';
 
 @Component({
-  selector: 'at-login',
```

Comment:
I removed a bunch of selectors that don't do anything. They're routed directly via routers.